### DISCUSSION TITLE
Wi-Fi Open Network Authentication

### DISCUSSION THREAD

**Discussion by nickcowley** [Blackymas/NSPanel_HA_Blueprint#751] [Q&A]
Hello

I have an open Wi-Fi SSID but seem unable to access once he NSPanel is flashed. OI can see the Wi-Fi icon on the panel, but not connecting.

I removed the   wifi_password: "" line so it deployed, but not sure if that is the issue.

Fo no password networks what should I set here?

Many thanks in advance.

**Comment by edwardtfn** [comment]
I know a baby could be made with the time since you asked this, but did you got your panel connected?
If you still wanna try this, I will be happy to help.
